frb_codegen/src/library/commands/cargo_expand/real.rs

```diff
@@ -55,16 +58,24 @@ fn run_raw(
     interest_crate_name: Option<&CrateName>,
     extra_rustflags: &str,
     allow_auto_install: bool,
+    features: Option<&[String]>,
 ) -> Result<String> {
     // let _pb = simple_progress("Run cargo-expand".to_owned(), 1);
     debug!("Running cargo expand in '{rust_crate_dir:?}'");
 
-    let args_choosing_crate = if let Some(interest_crate_name) = interest_crate_name {
+    let mut args_choosing_crate = if let Some(interest_crate_name) = interest_crate_name {
         vec!["-p", interest_crate_name.raw()]
     } else {
         vec![]
     };
 
+    if let Some(features) = features {
+        for feature in features {
+            args_choosing_crate.push("--features");
+            args_choosing_crate.push(feature.as_str());
+        }
+    }
```

Comment:
Looks like this is not `args_choosing_crate` - it is more like `args_features`. So maybe

```rust
let args_features = features.unwrap_or_default().iter()
.flat_map(|feature| vec!["--features", feature])
.collect_vec();
```